plugins/controlboard/src/ControlBoardDriverDeviceDriver.cpp

```diff
@@ -21,7 +21,6 @@ bool GazeboYarpControlBoardDriver::open(yarp::os::Searchable& config)
     m_deviceName = m_pluginParameters.find("name").asString().c_str();
 
     std::string robotName(m_pluginParameters.find("robotScopedName").asString().c_str());
-
```

Comment:
In general, please try to avoid touching files that are not related to the PR, as it complicates the reviewer life. If you want to cleanup a file, please do it in a separate PR or at least in a separate commit. Having said that, for this PR feel free to leave this change in, thanks!